doc/upgrading.md

```diff
@@ -21,26 +21,33 @@ See the [NEWS](../NEWS.md) summary of changes for each version.
 
 ## Version 33.0
 
+#### [RMB-862](https://issues.folio.org/browse/RMB-862) Upgrade to Vert.x 4.1.2
 
-#### [RMB-851](https://issues.folio.org/browse/RMB-851) Upgrade to Vert.x 4.1.0
-
-Module should depend on that version or a later version in 4.1.0 series.
+Module should depend on Vert.x 4.1.2 or a later version in 4.1.x series.
```

Comment:
RMB 33 has already been released and used by others. Should this change be included/mentioned in next release(33.1 at least)?

Reply:
Yes, good catch, thanks!
I've moved this into a new section "Version 33.1"